docs/UserGuide.md

```diff
@@ -70,16 +70,16 @@ good to take note of them before continuing.
 8. **Ascending**: Related to order; from low to high or minimum to maximum.
 9. **Descending**: Related to order; from high to low or maximum to minimum. 
 
-### Person-specific terms
-1. **Name**: The name of the person.
-2. **Phone**: The phone number of the person.
-3. **Email**: The email of the person.
-4. **Address**: The address of the person.
-5. **Price Range**: The price range of properties that a person might consider buying. That is, any property whose price
+### Buyer-specific terms
+1. **Name**: The name of the buyer.
```

Comment:
Actually, isn't `Name` used for the `Owner` class as well?

Reply:
Changed `Owner` to `Owner Name`!